var/spack/repos/builtin/packages/py-pyonmttok/package.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyPyonmttok(CMakePackage, PythonExtension):
+    """Tokenizer is a fast, generic, and customizable text tokenization library for
+    C++ and Python with minimal dependencies."""
```

Comment:
Seems like the wrong description

Reply:
Nope, that's accurate: https://github.com/OpenNMT/Tokenizer/